infra/postdeployment.tf

```diff
@@ -14,126 +14,132 @@
  * limitations under the License.
  */
 
-resource "google_compute_network" "gce_init" {
-  count = var.init ? 1 : 0
-
-  name                            = var.random_suffix ? "gce-init-network-${random_id.suffix.hex}" : "gce-init-network"
-  auto_create_subnetworks         = false
-  routing_mode                    = "GLOBAL"
-  project                         = var.project_id
-  delete_default_routes_on_create = false
-  mtu                             = 0
 
-  depends_on = [module.project_services]
+# used to collect access token, for authenticated POST commands
+data "google_client_config" "current" {
 }
 
-resource "google_compute_subnetwork" "gce_init" {
-  count = var.init ? 1 : 0
-
-  name          = var.random_suffix ? "subnet-gce-init-${random_id.suffix.hex}" : "subnet-gce-init"
-  network       = google_compute_network.gce_init[0].id
-  ip_cidr_range = "10.10.10.0/24"
-  region        = var.region
+# Job that uses pre-built docker image to deploy a placeholder website.
+resource "google_cloud_run_v2_job" "placeholder" {
+  name     = var.random_suffix ? "placeholder-${random_id.suffix.hex}" : "placeholder"
+  location = var.region
+
+  labels = var.labels
+
+  template {
+    template {
+      service_account = google_service_account.client.email
+      max_retries     = 1
```

Comment:
Should max_retries be boosted for improved reliability?

Reply:
I was using the default value for jobs; I moved this 'worker' job to this file, so it looks new; have reverted this setting.